Doppler.Currency/Doppler.Currency.csproj

```diff
@@ -9,6 +9,7 @@
   <ItemGroup>
     <PackageReference Include="AngleSharp" Version="0.14.0" />
     <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="3.1.8" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.8" />
```

Comment:
It's necessary to add NewtonsoftJson?

Reply:
Naza, it package is necessary for the hide default value, equaly, now I will unistall it package because the I will change the SaleValue as nulleable